bff/pkg/server/server.go

```diff
@@ -319,67 +339,72 @@ func metrics_handler(route_configs map[string]RouteConfig, dbIdentifiers []strin
 	})
 }
 
-func activity_handler(metrics_service metrics.Service) http.HandlerFunc {
+type ActivityParams struct {
+	DbIdentifier string `validate:"required,dbIdentifier,max=50"`
```

Comment:
Here is the limits for some fields:
- SystemID: for AWS it's min 1 character and max 63 characters. Let's use that for now.
- SystemScope:  it's <AWS_REGION>/<CLUSTER_PREFIX><AWS_ACCOUNT_ID> where AWS_REGION is between 1 and 50 characters, CLUSTER_PREFIX is between 0 and 11 characters, and AWS_ACCOUNT_ID is 12 characters. Then, the whole SystemScope is betwen 13 and 73 characters.
- SystemType: Here is the list of accepted values:
```
amazon_rds
google_cloudsql
azure_database
heroku
crunchy_bridge
aiven
tembo
self_hosted
```
- DBIdentifier = SystemID+SystemScope+SystemType

Reply:
Please document this in the code, too.